3.9.1/index.d.ts

```diff
@@ -57,8 +23,23 @@ declare interface Gulp {
    * </ul>
    */
   task: typeof Orchestrator.prototype.add;
-  run: RunMethod;
-  watch: typeof watch;
+  /**
+   * Run the specified task(s)
+   * @deprecated will be removed in gulp 4.0. Use task dependencies instead.
+   */
+  run: typeof Orchestrator.prototype.start;
+  /**
+   * Run the specified task(s)
+   * @deprecated will be removed in gulp 4.0. Use task dependencies instead.
+   */
+  start: typeof Orchestrator.prototype.start;
+  /**
+   * watches all files in the specified glob(s) for changes and runs the given task(s).
+   * @param globs
+   * @param tasks
+   */
+  watch(globs?: string | Array<string>, tasks?: task | Array<task>): EventEmitter;
+  watch(globs?: string | Array<string>, opt?: WatchOptions, tasks?: task | Array<task>): EventEmitter;
```

Comment:
I personally don't care so much about what style is being used, but I care about consistency, so if a project uses a certain style this style should be used consistently